### ISSUE TITLE
Can't import photos

### ISSUE THREAD

**Issue by pastarunner** [inaturalist/iNaturalistReactNative#2310]
**Describe the bug**
I'm not able to import photos to upload them as observations. Taking a photo with the app works fine but importing existing photos does not work.

**To Reproduce**
1. Tap on Green Plus at bottom and tap the photo icons that appear on the top right of the Green Plus.
3. Select any photo from your files, gallery, google photos, etc.
4. It gets stuck at a blank screen with green activity bar on top.

**Expected behavior**
I expected the photo to appear so it can be uploaded

**Screenshots**
![pulse-image-1730086901839](https://github.com/user-attachments/assets/a772e435-a690-4b4d-8845-3dc605ab6080)
![pulse-image-1730086911256](https://github.com/user-attachments/assets/73a7b860-8db6-4f6f-9664-eae93fbb6b44)
![pulse-image-1730086907325](https://github.com/user-attachments/assets/04c7dbe7-bf39-4dc7-a50f-c3262477d4db)
![pulse-image-1730086913452](https://github.com/user-attachments/assets/40ff3e45-e309-4a36-b9ee-455b5ee9140d)

**Context (please complete the following information):**
 - Device: Samsung Galaxy A54 5G
 - OS: Android version 14
 - App version: 0.52.0 (121)

**Comment by kueda** [comment]
This is probably a duplicate of #947, but since it's definitely working both online and offline, I guess we'll leave it. Both probably need the same solution.

**Comment by pastarunner** [comment]
I did see #947. That user mentioned it worked with screenshots but I can't make it work with anything, including screenshots. Also for note; I tried it with and without internet, and I tried with wifi and mobile data. Nothing

**Comment by shroomsnjunk** [comment]
This has been my issue since trying the app about a year ago as well. Every time an update is released I check to see if uploading works, but it never has for me, with or without internet connection it just loads infinitely. I'm using Android 14 as well

**Comment by stragu** [comment]
Same for me, can't create an observation from gallery photos, and can't add photos from gallery to an existing observation.

Using:

* iNat Next 0.58.2 (141)
* Galaxy A20e
* Android 11 (no further updates available for this device)

**Comment by danrademacher** [comment]
Yes. Sorry that note didn't come through when I closed the ticket. Since #2907 has the same issue recurring with a newer version of the app, I chose to stick with that one. When we get to refocus on Android (later this spring), we'll start with 2907. We can always circle back to this ticket if we need to. For scoping/estimating work, it's helpful to consolidate to fewer tickets, so long as we retain connections between tickets if we need to resurrect an apparent duplicate
